onnx/backend/test/case/node/groupnormalization.py

```diff
@@ -20,20 +20,20 @@ def _group_normalization(x, num_groups, scale, bias, epsilon=1e-5):
     axes = tuple(range(2, len(new_shape)))
     mean = np.mean(x_reshaped, axis=axes, keepdims=True)
     var = np.var(x_reshaped, axis=axes, keepdims=True)
-    dim_ones = (1,) * (len(new_shape) - 2)
+    x_normalized = ((x_reshaped - mean) / np.sqrt(var + epsilon)).reshape(x.shape)
+    dim_ones = (1,) * (len(x.shape) - 2)
     scale = scale.reshape(-1, *dim_ones)
     bias = bias.reshape(-1, *dim_ones)
-    res = scale * (x_reshaped - mean) / np.sqrt(var + epsilon) + bias
-    return res.reshape(x.shape)
+    return scale * x_normalized + bias
 
 
 class GroupNormalization(Base):
     @staticmethod
     def export() -> None:
         x = np.random.randn(3, 4, 2, 2).astype(np.float32)
         num_groups = 2
-        scale = np.random.randn(num_groups).astype(np.float32)
-        bias = np.random.randn(num_groups).astype(np.float32)
+        scale = np.random.randn(4).astype(np.float32)
```

Comment:
Maybe using another symbolic name (for the value 4) would help clarify things to a reader?

Reply:
Eg. use `C` is this is supposed to be C.